src/OrchardCore.Modules/OrchardCore.Email/Models/SmtpSettings.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.Options;
+using OrchardCore.Entities;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Settings;
+
+namespace OrchardCore.Email
+{
+    public class SmtpSettingsConfiguration : IConfigureOptions<SmtpSettings>
+    {
+        private readonly ISiteService _site;
+        private readonly IDataProtectionProvider _dataProtectionProvider;
+        private readonly ShellSettings _shellSettings;
+
+        public SmtpSettingsConfiguration(ISiteService site,
+            IDataProtectionProvider dataProtectionProvider,
+            ShellSettings shellSettings)
+        {
+            _site = site;
+            _dataProtectionProvider = dataProtectionProvider;
+            _shellSettings = shellSettings;
+        }
+
+        public void Configure(SmtpSettings options)
+        {
+            var settings = _site.GetSiteSettingsAsync()
+                .GetAwaiter().GetResult()
+                .As<SmtpSettings>();
+
+            options.DefaultSender = settings.DefaultSender;
+            options.Host = settings.Host;
+            options.Port = settings.Port;
+            options.EnableSsl = settings.EnableSsl;
+            options.RequireCredentials = settings.RequireCredentials;
+            options.UseDefaultCredentials = settings.UseDefaultCredentials;
+            options.UserName = settings.UserName;
+            // decrypt the password
+            if ( !String.IsNullOrWhiteSpace( settings.Password ) )
```

Comment:
Is there a way for you to apply the editor settings file we have in the root folder? What code editor are you using?

Reply:
I am using VS for Mac.  I guess the spaces were remnants when I had the custom style set in VS in windows.